english/let-me-out/let-me-out.md

```diff
@@ -0,0 +1,16 @@
+You take a moment to collect yourself. 
+Who wouldn't after all the craziness that has happened?
+"Doctor, you have to let me out. I'm not crazy."
+The doctor looks at you, skeptically.
+
+"You should know how this works by now," he sighs.
+And continues, "My hands are tied. There's nothing I can do."
+
+You notice that he is hiding something behind his back.
+
+Do you:
+
+[Kick the Doctor](kicktheman/kicktheman.md)
+[Charge for the lights, turn them off](lightsoff/lightsoff.md)
```

Comment:
There is a link on this line and also on the previous line.  Please make sure to insert a blank line between links.  That will help keep options visually separated.